config/config.yaml

```diff
@@ -0,0 +1,123 @@
+# This file contains defaults for the "config" object used in the Snakefile.
+# To temporarily override or provide a value, you can use snakemake's --config
+# or --configfile options.
+---
+s3_staging_url: s3://nextstrain-staging
+slack_token: ~
+slack_channel: "#ncov-gisaid-updates"
+sequences: "data/sequences.fasta"
+metadata: "data/metadata.tsv"
+
+reference_node_name: "USA/WA1/2020"
+
+# Define files used for external configuration. Common examples consist of a
+# list of strains to include and exclude from analyses, a reference sequence to
+# align sequences to, colors for strain attributes in auspice, and auspice
+# configuration files.
+files:
+  include: "config/include.txt"
+  exclude: "config/exclude.txt"
+  reference: "config/reference.gb"
+  outgroup: "config/outgroup.fasta"
+  ordering: "config/ordering.tsv"
+  color_schemes: "config/color_schemes.tsv"
+  auspice_config: "config/auspice_config.json"
+  auspice_config_gisaid: "config/auspice_config_gisaid.json"
```

Comment:
Can we move files like these somewhere where they only are listed for 'our' runs? Or is that too complicated? I just worry as with the above that people tend to get really caught up in this kind of thing, and these are files that really only we have to worry about. 
However, this is a fairly minor concern.